tests/src/dir.props

```diff
@@ -55,7 +55,7 @@
     <AddAdditionalExplicitAssemblyReferences>false</AddAdditionalExplicitAssemblyReferences>
     <GenerateTargetFrameworkAttribute>false</GenerateTargetFrameworkAttribute>
     <!-- Disable some C# warnings for the tests. -->
-    <NoWarn>78,162,164,168,169,219,251,252,414,429,642,649,652,675,1691,1717,1718,3001,3002,3003,3005,3008</NoWarn>
+    <NoWarn>78,162,164,168,169,219,251,252,414,429,618,642,649,652,675,1685,1691,1717,1718,3001,3002,3003,3005,3008</NoWarn>
```

Comment:
We shouldn't hide CS0618, it is listing code being used that was marked `Obsolete`

Reply:
I agree with this for production code, but I think it is fine to disable it for tests.

We need to have tests for obsolete APIs. It is pretty annoying to disable the warning individually around every test that is testing obsolete API.

Reply:
Agreed. Testing code has different requirements with respect to code marked `Obsolete`. Regardless if it is marked that way or not users can use the API and as such it must be tested.